src/UnitTest/tests/Base/TestData.h

```diff
@@ -1291,10 +1291,92 @@ void Data_object::test< 25 >() {
          std::vector< tk::real >{ 0.1, 0.1, 0.1 }, e2.extract( 0, 1 ) );
 }
 
-//! Test tk::Data's operator/=
+//! Test tk::Data's operator*= by tk::real as rhs
 //! \author J. Bakosi
 template<> template<>
 void Data_object::test< 26 >() {
```

Comment:
For this and all other test objects: Could we avoid the use of magic numbers? In this case it is fairly simple to map back and forth but if we don't need them, let's not use them. It will increase the clarity of the code and avoid the inevitable errors/strange behavior that will occur when there are multiple coders or someone doesn't have a whole picture of all the tests.